src/buildHTML.js

```diff
@@ -688,6 +688,7 @@ groupTypes.accent = function(group, options) {
             // So now we use an SVG.
             // If Safari reforms, we should consider reverting to the glyph.
             accent = buildCommon.staticSvg("vec", options);
+            accent.width = parseFloat(accent.style.width);
```

Comment:
This is kind of hacky, but since it should disappear once we have non-combining `\vec` I'm okay with this.